sale_order_safe_commitment_date/views/sale_order_views.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <sheet position="before">
+                <field name="is_commitment_date_unsafe" invisible="1" />
+                <div
+                    class="alert alert-warning"
+                    attrs="{'invisible': [('is_commitment_date_unsafe', '=', False)]}"
+                    role="alert"
+                >
+                    The delivery date is set before the expected date. At the order confirmation, the expected date (<field
+                        name="expected_date"
+                        readonly="1"
+                        nolabel="1"
+                    />) will be forced in order to fulfill the deliveries. Contact a sales manager to change the expected date in the deliveries.
+                </div>
+            </sheet>
+        </field>
+    </record>
+    <record id="view_order_form_commitment_date_ux" model="ir.ui.view">
+        <field name="model">sale.order</field>
+        <field
+            name="inherit_id"
+            ref="sale_order_safe_commitment_date.view_order_form"
+        />
+        <field name="arch" type="xml">
+            <field name="show_update_pricelist" position="before">
+                <label for="commitment_date" string="Delivery Date" />
+                <div name="commitment_date_div" class="o_row">
+                    <field
+                        name="commitment_date"
+                        attrs="{'invisible': [('commitment_date', '=', False), ('state', 'not in', ('draft', 'sent'))]}"
+                    />
+                    <span name="expected_date_span" class="text-muted"><i
+                            class="fa fa-warning text-warning"
+                            attrs="{'invisible': [('is_commitment_date_unsafe', '=', False)]}"
+                            alt="The delivery date is sooner than the expected date. You may be unable to honor the delivery date."
+                        /> Expected: <field
+                            name="expected_date"
+                            class="oe_inline"
+                            widget="date"
+                        /></span>
+                </div>
+            </field>
+        </field>
+    </record>
+    <data noupdate="1">
+        <!-- TODO: Enable via res.config.settings -->
+        <record id="view_order_form_commitment_date_ux" model="ir.ui.view">
+            <field name="active" eval="False" />
+        </record>
+    </data>
```

Comment:
Why is this necessary?

Reply:
Just go to the views and unarchive it?
How can be enabled via res.config.settings?
Can we add CONFIGURE.md to handle this situation?